docs/tutorials/migrate-to-mui5.md

```diff
@@ -0,0 +1,35 @@
+---
+id: migrate-to-mui5
+title: Migrating from Material UI v4 to v5
+description: Additionally resources to the Material UI migration guide specifically for Backstage
+---
+
+Before diving into the specific changes in Backstage, it may be helpful to take a look at the [Migration Guide provided by MUI](https://mui.com/material-ui/migration/migration-v4/). It breaks down the differences between v4, v5 and will make it easier to understand the impact on your Backstage instance & plugins.
```

Comment:
```suggestion
Before diving into the specific changes in Backstage, it may be helpful to take a look at the [Migration Guide provided by MUI](https://mui.com/material-ui/migration/migration-v4/). It breaks down the differences between v4 and v5, and will make it easier to understand the impact on your Backstage instance & plugins.
```